hpOneView/resources/security/certificates_server.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from future import standard_library
+
+standard_library.install_aliases()
+
+
+from hpOneView.resources.resource import Resource
+
+
+class CertificatesServer(Resource):
+    """
+    The Certificates Servers resource provides REST APIs for configuration of device or
+    server certificates for the appliance to establish SSL communication with other managed network entities.
+
+    Import, Update and Delete APIs are asynchronous and GET API is synchronous.
+
+    """
+    URI = '/rest/certificates'
+
+    DEFAULT_VALUES = {
+        '600': {"type": "CertificateInfoV2"},
```

Comment:
we don't need 600 if we are supporting from 800 onwards

Reply:
Removed API 600